Index.html

```diff
@@ -9,28 +9,23 @@
 </head>
 <body>
    <div class="container-home">
-       <div class="text-block">
            <h4>made with love</h4>
            <p>for you</p>
            <div>
                <button class="button"><a class="arrow">&#8249;</a></button>
                <button class="button"><a class="arrow">&#8250;</a></button>
-           </div>
-       </div>
+           </div> 
    </div>
-   &nbsp;
-   <div>
+   <div class="container-image"   >
        <img class="backgroud-image" src="/images/about_img_2.png">
        <img class="top-image" src="/images/about_img_1.png">
    </div>
-   &nbsp;
-   <div>
+   <div class="container-about">
        <h1>about</h1>
        <h5>WE CREATE DELICIOUS MEMORIES</h5>
```

Comment:
Also you should hardcode it here with uppercase letters, but do it with css instead